CHANGELOG.md

```diff
@@ -166,4 +168,4 @@ file an issue, so that we can see how to handle this.
   those tests have a probability > 0 for failing.
   If you want to run them anyways, pass
   `-DENABLE_GEANT3_TESTING=ON` to CMake.
-* CMake: added `BUILD_DATAMATCH` build switch, default is `ON`.
+* CMake: added `BUILD_DATAMATCH` build switch, default is `OFF`.
```

Comment:
So, what is now the plan? The first commit is released with 18.8? The second commit with 19.0? If yes, then why would a change in the 19.0 release modify the changelog text of a previous release?